I/Infernal/build_tarballs.jl

```diff
@@ -0,0 +1,74 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, BinaryBuilderBase, Pkg
+
+name = "Infernal"
+version = v"1.1.4"
+
+easel_version = v"0.48"
+hmmer_version = v"3.3.2"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/EddyRivasLab/infernal/archive/refs/tags/infernal-$(version).tar.gz",
+                  "311163c0a21a216e90862df81cd6c13811032076535e44fed4eabdcf45093fea"),
+    ArchiveSource("https://github.com/EddyRivasLab/easel/archive/refs/tags/easel-$(easel_version.major).$(easel_version.minor).tar.gz",
+                  "c5d055acbe88fa834e81424a15fc5fa54ac787e35f2ea72d4ffd9ea2c1aa29cf"),
+    ArchiveSource("https://github.com/EddyRivasLab/hmmer/archive/refs/tags/hmmer-$(hmmer_version).tar.gz",
+                  "fab109c67fb8077b32f7907bf07efbc071147be0670aee757c9a3ca7e2d485be")
```

Comment:
Why these are extra sources instead of their own JLL packages?

Reply:
JLL packages for easel and hmmer definitely make sense, but their build scripts don't produce any shared libraries and the static libraries that are built are not installed by `make install`.

In the Infernal build, hmmer and easel are statically linked.

I guess it would be possible to install the static libraries in the easel and hmmer jll packages, and then use the static libs during the infernal build, though i think the infernal build expects the hmmer and easel full source directories.

Reply:
In the (infernal-1.1.4 release)[http://eddylab.org/infernal/infernal-1.1.4.tar.gz] from the (infernal website)[http://eddylab.org/infernal/] hmmer and easel are included in source form. Should i use that tarball instead?

Reply:
That'd be nicer, so we don't have multiple sources, but since they want the source code and link those libraries statically, not having separate JLL packages makes sense, so I merged the current PR already.